app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,12 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.Recipeopedia"
         tools:targetApi="31">
+        <activity
+            android:name=".RecipeDetailsActivity"
+            android:exported="true" />
+        <activity
+            android:name=".RecipeListActivity"
+            android:exported="false" />
```

Comment:
fwiw, `exported` attribute defaults to false, you probably don't need to specify it at all